### DISCUSSION TITLE
Stable versions for PyG?

### DISCUSSION THREAD

**Discussion by hkim716** [pyg-team/pytorch_geometric#3963] [Q&A] [answered]
My codes were mixed up due to different versions of ```torch```, ```cuda```, and ```torch_geometric```.
Currently my computer supports ```cuda 11.3```, and it makes me use of ```torch 1.10``` and newer version of ```torch_geometric```. 
I think current versions of PyG are not that stable, having some incompatibilities in the libraries. 
Which version would you recommend to use the most stable PyG environment?

**Comment by rusty1s** [comment, marked as answer]
What do you mean with PyG not being stable? It definitely should be :) Any pointers highly appreciated.

If you do not want to make use of PyG 2.0, `pip install torch-geometric==1.7.2` should work as well on PyTorch 1.10 and CUDA 11.3.
